sonic.asm

```diff
@@ -54,34 +102,38 @@ loc_E0:
 		jmp	(loc_D5A2).l
 
 		dc.w ErrorTrap
-		dc.l ErrorTrap, ErrorTrap, ErrorTrap
+		dc.l ErrorTrap
+		dc.l ErrorTrap
+		dc.l ErrorTrap
 	endif
-Console:	dc.b "SEGA MEGA DRIVE " ; Hardware system ID
-Date:		dc.b "(C)SEGA 1991.APR" ; Release date
+Console:	dc.b "SEGA MEGA DRIVE " ; Hardware system ID (Console name)
+Date:		dc.b "(C)SEGA 1991.APR" ; Copyright holder and release date (generally year)
 Title_Local:	dc.b "SONIC THE               HEDGEHOG                " ; Domestic name
 Title_Int:	dc.b "SONIC THE               HEDGEHOG                " ; International name
 Serial:		if Revision=0
-		dc.b "GM 00001009-00"   ; Serial/version number
+		dc.b "GM 00001009-00"   ; Serial/version number (Rev 0)
 		else
-			dc.b "GM 00004049-01"
+			dc.b "GM 00004049-01" ; Serial/version number (Rev non-0)
 		endc
-Checksum:	dc.w 0
+Checksum:	dc.w $264A
 		dc.b "J               " ; I/O support
-RomStartLoc:	dc.l StartOfRom		; ROM start
-RomEndLoc:	dc.l EndOfRom-1		; ROM end
-RamStartLoc:	dc.l $FF0000		; RAM start
-RamEndLoc:	dc.l $FFFFFF		; RAM end
+RomStartLoc:	dc.l StartOfRom		; Start address of ROM
+RomEndLoc:	dc.l EndOfRom-1		; End address of ROM
+RamStartLoc:	dc.l $FF0000		; Start address of RAM
+RamEndLoc:	dc.l $FFFFFF		; End address of RAM
 SRAMSupport:	if EnableSRAM=1
 		dc.b $52, $41, $A0+(BackupSRAM<<6)+(AddressSRAM<<3), $20
 		else
 		dc.l $20202020
 		endc
 		dc.l $20202020		; SRAM start ($200001)
 		dc.l $20202020		; SRAM end ($20xxxx)
-Notes:		dc.b "                                                    "
-Region:		dc.b "JUE             " ; Region
+Notes:		dc.b "                                                    " ; Notes (unused, anything can be put in this space, but it has to be 52 bytes.)
+Region:		dc.b "JUE             " ; Region (Country code)
+EndOfHeader:
 
 ; ===========================================================================
+; Crash/Freeze the 68000. Note that the Z80 continues to run, so the music keeps playing.
```

Comment:
Sonic 1 plays music on the 68000, so "the music keeps playing" is wrong.

Reply:
oops changing that now